app/container-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eu
+
+bin='docker-gen'
+
+# run command if it is not starting with a "-" and is an executable in PATH
+if [ "${#}" -le 0 ] || \
```

Comment:
`command` will return 1 if the command name is the empty string, so this check isn't necessary.